scipy/signal/ltisys.py

```diff
@@ -1337,7 +1337,7 @@ def __init__(self, *system, **kwargs):
             return
 
         # Remove system arguments, not needed by parents anymore
-        super(StateSpace, self).__init__(**kwargs)
```

Comment:
Can it be that due to the possible inheritence change in `__new__` then the super here could call something different than `StateSpace`?

Same comment for the other `__init__` from this file.